client/package.json

```diff
@@ -3,6 +3,6 @@
     "license": "GPL-3.0",
     "dependencies": {
         "angular-sanitize": "^1.6.7",
-        "superdesk-core": "superdesk/superdesk-client-core#master"
+        "superdesk-core": "superdesk/superdesk-client-core#8e4ea69"
```

Comment:
shouldn't this point to master when the PR is merged anyway?

Reply:
It should. I wanted to make a working instance in fireq.